mongobetween.go

```diff
@@ -21,11 +22,17 @@ func main() {
 func run(config *config.Config) {
 	proxies, err := config.Proxies(config.Logger())
 	log := config.Logger()
-
 	if err != nil {
 		log.Fatal("Startup error", zap.Error(err))
 	}
 
+	profiler := util.StartDatadogProfiler(log)
+	if profiler != nil {
+		defer func() {
+			profiler.Stop()
+		}()
+	}
```

Comment:
We should gate this behind a flag -- I haven't run benchmarks myself, but I'm worried there's a non-zero impact to performance.